staging/object-bucket-claim/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: object-bucket-claim
+description: A Helm chart for ObjectBucketClaim to be consumed by kommander
```

Comment:
Can we genericize this description to not be specific to kommander? Maybe some mention of why we are wrapping the CRD in a chart to allow configuration, etc. This chart is public and can be used by external users if they wanted to use it. I would also move this into `stable/` dir, since this isn't a forked chart.. I just noticed we actually have a definition of what should be stable or staging https://github.com/mesosphere/charts#repository-structure I think this particular chart fits more under `stable`.

Reply:
done